includes/apple-push-api/request/class-request.php

```diff
@@ -240,13 +240,16 @@ private function parse_response( $response, $json = true, $type = 'post', $meta
 
 			// Add the MIME request.
 			$body .= "\n\n" . esc_html__( 'MIME request', 'apple-news' ) . ":\n" . $debug_mime_request . "\n";
+			
+			$headers = apply_filters( 'apple_news_notification_headers', '' );
```

Comment:
Could you add a docblock to document the filter à la https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters please?

Reply:
I've added a docblock, as requested. In another ticket, it would be good to add docblocks for other filters in this file.

Reply:
Yes, I agree. We've been pretty consistent about adding them in recent code, but haven't gone back and added them universally in the older parts of the codebase.